### PR TITLE
initial infobox service structure

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: service/infobox/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+ExampleConfiguration - Contains whatever you want.
+Data here are created from example.yaml by magic(reflection)
+ 
+ExampleResource - Controller handling your requests. 
+Contains entry points, and logic what, and how service should return
+
+ExampleHealthCheck - Self explanation class
+
+banner.txt - [Why this is important](https://dropwizard.github.io/dropwizard/manual/core.html#banners)
```
Comment (pchojnacki) [inline service/infobox/README.md]: you can cleanup the README
You can give good example to others and also add some brief description of what the service will be doing :D

File: service/infobox/src/main/java/com/wikia/infobox/service/resources/ParserResource.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  @Inject
+  public ParserResource(InputNormalizer normalizer, Map<String, OutputFormatter> formatters) {
+    this.normalizer = normalizer;
+    this.formatters = formatters;
+  }
+
+  @POST
+  @Produces(MediaType.TEXT_HTML)
+  public Response parseToHtml(String data) {
+    System.out.println(data);
```
Comment (pchojnacki) [inline service/infobox/src/main/java/com/wikia/infobox/service/resources/ParserResource.java]: I know its temporary, but I'd prefer not to LGTM a commit with very visible system out println.

File: service/infobox/src/main/resources/banner.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+███████╗███████╗██████╗ ██╗   ██╗██╗ ██████╗███████╗
+██╔════╝██╔════╝██╔══██╗██║   ██║██║██╔════╝██╔════╝
+███████╗█████╗  ██████╔╝██║   ██║██║██║     █████╗
+╚════██║██╔══╝  ██╔══██╗╚██╗ ██╔╝██║██║     ██╔══╝
+███████║███████╗██║  ██║ ╚████╔╝ ██║╚██████╗███████╗
+╚══════╝╚══════╝╚═╝  ╚═╝  ╚═══╝  ╚═╝ ╚═════╝╚══════╝
```
Comment (pchojnacki) [inline service/infobox/src/main/resources/banner.txt]: If I didn't knew this should represent Doge then I would think its a snowman or something similar :P

**Comment by pchojnacki** [comment]
It seems like its mostly boilerplate code for a new service so not much to review here.
So provisional - LGTM

**Comment by sqreek** [comment]
Code seems good, but we change dropwizard-guice for dropwizard-guicey.
Check [this](https://github.com/Wikia/pandora/blob/master/service/mobile-config/src/main/java/com/wikia/mobileconfig/MobileConfigApplication.java) for more info
There is only method rename and LGTM

**Comment by pchojnacki** [comment]
I'll be refactoring that too in my PR so for now its ok to commit that IMO. I'm not sure if the new guice bundle and changes still make it easy to stub out the Consul for local development

**Pull request merged**